<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CoinBase</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/money.png</normaloff>images/money.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>choose currency to convert from</string>
    </property>
    <item>
     <property name="text">
      <string>Egyptian Pound</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/egypationPound.png</normaloff>images/egypationPound.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>United States Dollar </string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/dollar.png</normaloff>images/dollar.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Euro</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/euro.png</normaloff>images/euro.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>indian rupee</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/irupee.png</normaloff>images/irupee.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>japanese yen</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/japanese-yen.png</normaloff>images/japanese-yen.png</iconset>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>choose currency to convert from</string>
    </property>
    <item>
     <property name="text">
      <string>United States Dollar </string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/dollar.png</normaloff>images/dollar.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Euro</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/euro.png</normaloff>images/euro.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Egyptian Pound</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/egypationPound.png</normaloff>images/egypationPound.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>indian rupee</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/irupee.png</normaloff>images/irupee.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>japanese yen</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/japanese-yen.png</normaloff>images/japanese-yen.png</iconset>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="result_button">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>100</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Click to show the result</string>
    </property>
    <property name="text">
     <string>esult</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/result.png</normaloff>images/result.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Enter The amount</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>280</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
